easybuild/tools/variables.py

```diff
@@ -52,7 +52,7 @@ def get_class(name, default_class, map_class=None):
     if name is not None:
         try:
             klass = map_class[name]
-        except:
+        except BaseException:
             for k, v in map_class.items():
```

Comment:
More single-letter variables that would be nice to get rid of...

Reply:
Unrelated